highcharts-behavior.js

```diff
@@ -0,0 +1,266 @@
+import * as async from '@polymer/polymer/lib/utils/async.js'
+
+import 'highcharts/highcharts.js';
+import 'highcharts/highcharts-more.js';
+
+Highcharts.setOptions({global: {useUTC: false}});
+
+// from jQuery
+function isFunction(obj) {
```

Comment:
As of now your code will not work!

```suggestion
const isFunction = o => typeof obj === "function" && typeof obj.nodeType !== "number"
```